soh/soh/Enhancements/randomizer/3drando/item_pool.cpp

```diff
@@ -798,6 +798,15 @@ void GenerateItemPool() {
     AddItemToMainPool(GOLD_SKULLTULA_TOKEN, 100);
   }
 
+  if (Shuffle100GSReward) {
+    if (ItemPoolValue.Is(ITEMPOOL_PLENTIFUL)) {
+      AddItemToPool(PendingJunkPool, GOLD_SKULLTULA_TOKEN, 10);
```

Comment:
how does this work with tokensanity off? (i wouldn't expect many people to play with 100 skull reward and tokensanity off but it's possible)

Reply:
I completely forgot about tokensanity. I'll update this to check for it.